blobl-editor/wasm/main.go

```diff
@@ -1,206 +1,217 @@
 package main
 
 import (
-    "encoding/json"
-    "fmt"
-    "syscall/js"
+	"encoding/json"
```

Comment:
Somehow, this file had the tabs converted to spaces in a previous PR. Not sure why...